keep/api/core/db.py

```diff
@@ -3264,6 +3266,80 @@ def remove_alerts_to_incident_by_incident_id(
         return deleted
 
 
+class DestinationIncidentNotFound(Exception):
+    pass
```

Comment:
does it need to be here? and does it need to wrap HTTPException with status code 404?

Reply:
no strong opinion